aissemble-universal-config-loader/src/aissemble_universal_config_loader/config_loader.py

```
@@ -0,0 +1,94 @@
+
+    def get_property(self, file: str, property_key: str) -> str:
+        """
+        get the given property key value from the given
+        :param file: property file
+        :param property_key: property key
+        :return: value
+        """
+        return self._property_manager.get_properties(file).getProperty(property_key)
+
+    def load(
+        self,
+        file: str = "configuration.properties",
+        to_env: bool = False,
+        to_var: bool = False,
+        globals: dict = None,
+    ):
+        """
+        Load the .properties file from configurations/base directory
+        :param file: (Optional) If not set, the default property file name, configuration.properties, is used
+        :param to_env: (Optional) if set to True, the property values will be loaded as environment variables
+        :param to_var: (Optional) if set to True, the property values will be loaded as global variables
+        :param globals: (Optional) this parameter is only required when the to_var is set to True. In that case, it should be set to globals=globals().
+                                  This sets the scope of the globals() tables the variables should be loaded
+        :return properties
+        """
+
+        # only use BASE_PROPERTY as file path configuration if set
+        os.environ["KRAUSENING_BASE"] = os.environ.get("BASE_PROPERTY", "")
+
+        # only support base properties for now
+        if os.environ.get("KRAUSENING_EXTENSIONS", None) is not None:
+            os.environ.pop("KRAUSENING_EXTENSIONS")
+
+        # only support base properties for now
+        if os.environ.get("KRAUSENING_OVERRIDE_EXTENSIONS", None) is not None:
+            os.environ.pop("KRAUSENING_OVERRIDE_EXTENSIONS")
+
+        properties = self._property_manager.get_properties(file)
+
+        for property_key, property_value in properties.items():
+            # replace dash (-) with underscore (_)
+            property_key = property_key.replace("-", "_")
```

S: Document this behavior.  Also, does it need to be done for envvars? I'd think it would only matter for Python variable names. And I think it's probably OK to just let it fail if they have an incompatible name unless we think this will be a common case.

My thought is that if they're using `load_as_globals` then they'd expect to have to have compatible property names.